### PR TITLE
Loginandsignup

### PR DESCRIPTION
**Pull request by hjtrhodes**
Styling removed - login and signup functionality working but must be accessed from separate page /loginsignup

User currently will need to signup then login

### CODE REVIEW

File: frontend/src/components/LoginSignupModal/LoginSignupModal.jsx [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+            <input id="login-email" type="email" placeholder="Email" value={ email } onChange={handleEmailChange} />
+            <input id="login-password" type="password" placeholder="Password" value={ password } onChange={handlePasswordChange} />
+            {errorMessage && (
+            <div className="error-message">
+
+                <p>{errorMessage}</p>
+            </div>
+            )}
+
+            <button id="login-submit" type="submit">Submit</button>
```
Comment (pablisch) [inline frontend/src/components/LoginSignupModal/LoginSignupModal.jsx:136]: Could this button say something like 'Log in' to match the syntax on the sign up button? Marie will like this. Submit is very standard practice but also pretty meaningless in accessibility terms.

File: frontend/src/components/LoginSignupModal/LoginSignupModal.jsx [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+    event.preventDefault();
+
+    let response = await fetch(`${baseUrl}/api/auth/login`, {
+        method: 'POST',
+        headers: {
+        'Content-Type': 'application/json',
+    },
+        body: JSON.stringify({ email: email, password: password })
+    });
+    if (response.status === 200) {
```
Comment (pablisch) [inline frontend/src/components/LoginSignupModal/LoginSignupModal.jsx:38]: 201 is the standard expected code for a POST request as you have for the other request. Ignore this if that was intentional.

**Review by pablisch — DISMISSED** [review]
Great. I messaged separately about something I was unsure and then there are two tiny optional changes.

**Review by Mike-W00d — APPROVED** [review]
LGTM!

**Pull request merged**